<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddActionDialog</class>
 <widget class="QDialog" name="AddActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>78</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>78</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dodawanie akcji</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_NameLabel">
     <property name="text">
      <string>Nazwa:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="m_ActionName">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="m_Templates" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="m_TemplateName"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_ManageTemplates">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ZarzÄ…dzaj</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_ActionName</tabstop>
  <tabstop>m_TemplateName</tabstop>
  <tabstop>m_ManageTemplates</tabstop>
  <tabstop>m_ButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddActionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddActionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ManageTemplates</sender>
   <signal>clicked()</signal>
   <receiver>AddActionDialog</receiver>
   <slot>manageTemplates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>manageTemplates()</slot>
  <slot>dateTimeChanged(QDateTime)</slot>
 </slots>
</ui>
